includes/admin/class-wc-payments-admin.php

```diff
@@ -241,7 +241,7 @@ public function register_payments_scripts() {
 		$wcpay_settings = [
 			'connect'               => [
 				'url'                => WC_Payments_Account::get_connect_url(),
-				'country'            => wc_get_base_location()['country'],
+				'country'            => WC()->countries->get_base_country(),
```

Comment:
@kalessil I switched this for consistency with the other change, and because IMO it's a little bit cleaner. It's functionally the same though (except for this one having a filter applied) so feel free to object to this change for any reason.